src/main/frontend/components/block.cljs

```diff
@@ -238,18 +238,18 @@
           [:a.delete.ml-1
            {:title    "maximize image"
             :on-click (fn [^js e] (let [images (js/document.querySelectorAll ".asset-container img")
-                                        images (to-array images)
-                                        images (if-not (= (count images) 1)
-                                                 (let [^js _image (.closest (.-target e) ".asset-container")
-                                                       image (. _image querySelector "img")]
-                                                   (cons image (remove #(= image %) images)))
-                                                 images)
-                                        images (for [^js it images] {:src (.-src it)
-                                                                     :w (.-naturalWidth it)
-                                                                     :h (.-naturalHeight it)})]
-
-                                    (when (seq images)
-                                      (lightbox/preview-images! images))))}
+                                       images (to-array images)
```

Comment:
Is the editor setting changed? There shouldn't be so many whitespace changes.

Reply:
Not sure what happened in my editor :)
![CleanShot 2021-12-28 at 15 42 49](https://user-images.githubusercontent.com/479169/147541446-dcffdcfc-7d4f-4b67-b048-9985579bbb72.png)
Shall we force the code style through some code formatters in CI?

Reply:
`fancify-symbols` enabled?
the alignment is offset by 1, `fn` <-> `\lambda`

Reply:
Indeed, prettify-symbols-mode was enabled.